api/v2.0/swagger.yaml

```diff
@@ -9349,10 +9349,9 @@ definitions:
         format: int64
         description: The artifact id of the accessory
         x-omitempty: false
-      subject_artifact_id:
```

Comment:
This is a breaking change, removing functionality. 
I would prefer to support both options.

Reply:
Okay, that's a good point. I will submit another pull request to reinstate it. However, we should consider keeping it in, as subsequent pull requests depend on it.